common/auth/authorizer.js

```diff
@@ -8,30 +8,23 @@ const AuthGroups = require('./authGroups')
 const {
   permissions,
   keys,
-  getAuthGroups,
 } = AuthGroups
 
-const isSurveyAdmin = (user, surveyInfo) =>
+const _isSurveyAdmin = (user, surveyInfo) =>
   Survey.isAuthGroupAdmin(getSurveyUserGroup(user, surveyInfo))(surveyInfo)
 
 // ======
 // ====== Survey
 // ======
 
+const _getNonSystemAdminSurveyUserGroup = (user, surveyInfo) =>
```

Comment:
what about adding a boolean param e.g.:  ```getSurveyUserGroup(user, surveyInfo, includeSystemAdmin = true)```. so when you call it from below, you pass false ?
(Instead of adding this method)

Reply:
Ok